libopeniscsiusr/sysfs.c

```diff
@@ -454,8 +455,10 @@ bool _iscsi_transport_is_loaded(const char *transport_name)
 	return false;
 }
 
-int _iscsi_iface_kern_id_of_host_id(struct iscsi_context *ctx,
-				    uint32_t host_id, char *iface_kern_id)
+int _iscsi_iface_kern_ids_of_host_id(struct iscsi_context *ctx,
+				    uint32_t host_id,
+				    char ***iface_kern_ids,
+				    uint32_t *iface_count)
```

Comment:
@cleech Do you have any example or kernel code indicate their might be multiple ifaces binding to single scsi host?

Reply:
Both be2iscsi and qla4xxx (so, the only two users of the iscsi_create_iface API) will create both IPv4 and IPv6 ifaces depending on the firmware setting on the card.  I think you do need to enable IPv6 in the HBA firmware, because I know my qla4xxx test box was creating 3-per-host at one time and right now it's only doing 1.